### ISSUE TITLE
hello, I have a question, so adding it to the issue.

### ISSUE THREAD

**Issue by lllilllilllilili** [ZackAkil/video-intelligence-api-visualiser#7]
I read the source code carefully. thank you.

I fork the repository, configure the web front rendering with http, and then upload the video, the json file will not come out automatically. How can I automate this?

Should I configure these environments automatically? 

Thanks!

**Comment by dylancaponi** [comment]
No, the python does not run automatically, you need to trigger it.

I think you did things the way this project intends.
